config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -862,6 +862,9 @@ periodics:
       - --warnings=validate-urls
       - --warnings=unknown-fields
       - --warnings=duplicate-job-refs
+      - --warnings=unknown-fields-all
```

Comment:
unknown fields were fixed last year, but forgot to have them added to the ci job https://github.com/kubernetes/test-infra/pull/27584